UnitedIncome/components
# PaginationControls Atom - prioritizing the border-radius property

### Description
> Please provide a brief description of your changes below.

Ensuring that the border-radius property set the pagination controls is prioritized over any other rule set.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Ensure that the pagination controls buttons under Atoms > PaginationControls still have flattened corners(ie that they are not rounded)(see screenshot)

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

<img width="398" alt="Screen Shot 2020-06-04 at 1 56 41 PM" src="https://user-images.githubusercontent.com/61422320/83794108-769e0380-a66b-11ea-9cb7-a40e858aa52f.png">

### Comments
> Any additional comments that will aid with the review process.
